src/figuresClasses.ts

```diff
@@ -1,11 +1,84 @@
-export interface Figure {}
+export interface Figure {
+  getArea(): number;
+  shape: string;
+  color: string;
+}
+
+export class Triangle implements Figure {
+  public shape: string = 'triangle';
+
+  public color: string;
+
+  constructor(
+    color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.color = color;
+
+    const sides = [a, b, c].sort((x, y) => x - y);
+
+    if (sides[2] >= sides[0] + sides[1]) {
+      throw new Error(`Sides ${a}, ${b} and ${c} can't form a triangle`);
+    }
+  }
+
+  getArea = (): number => {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Number(area.toFixed(2));
+  };
+}
+
+export class Circle implements Figure {
+  public shape: string = 'circle';
 
-export class Triangle implements Figure {}
+  public color: string;
 
-export class Circle implements Figure {}
+  constructor(
+    color: string,
+    private radius: number,
+  ) {
+    this.color = color;
 
-export class Rectangle implements Figure {}
+    if (radius <= 0) {
+      throw new Error('Radius must be greater than 0');
```

Comment:
[CODE STYLE] - If you throw an error try to provide its detailed description in error message. In this case, you could provide more context about why the radius must be greater than 0. For example, you could say 'A circle's radius must be a positive number greater than 0'.